Client/Coordinators/PasswordManagerCoordinator.swift

```diff
@@ -71,18 +71,32 @@ class PasswordManagerCoordinator: BaseCoordinator,
     func pressedPasswordDetail(model: PasswordDetailViewControllerModel) {
         let viewController = PasswordDetailViewController(viewModel: model)
         viewController.coordinator = self
+        sendLoginsManagementLoginsTappedTelemetry()
```

Comment:
Q. Any reason we are adding a method to call a method, can we not just add directly telemetry wrapper here? 🤔

Reply:
my reasoning was that the telemetry method takes 3 lines and it would clutter the method they'd be called from and this way I could have all the telemetry methods organized in one place compared to having to hunt for the events all over the place, and the method name makes it easier to read what telemetry you're sending instead of reading the parameters, so overall just for making the class easier to read at a glance